src/azure-cli/azure/cli/command_modules/acs/_params.py

```diff
@@ -20,7 +20,7 @@
     validate_create_parameters, validate_k8s_client_version, validate_k8s_version, validate_linux_host_name,
     validate_list_of_integers, validate_ssh_key, validate_connector_name, validate_max_pods, validate_nodepool_name,
     validate_vm_set_type, validate_load_balancer_sku, validate_load_balancer_outbound_ips,
-    validate_load_balancer_outbound_ip_prefixes, validate_taints)
+    validate_load_balancer_outbound_ip_prefixes, validate_taints, validate_ip_ranges, validate_acr, validate_ip_ranges_on_create)
```

Comment:
Remove the validator "validate_ip_ranges_on_create" as its not defined.

Reply:
Or maybe it makes sense to have another method on_create which does not allow empty string - ""?

Reply:
Nice catch, removed this dangling reference. I think we can avoid having another method to catch the "" on create. Technically, if a customer sets that it is ok right? It is essentially enabling IPWL to a state where it doesn't restrict any traffic. 

If you feel strongly about adding it, I am open to it :)

Reply:
I think its fine. Can you just test this scenario that nothing unexpected is happening.

Reply:
Will run manually and let you know on teams